src/functionalTest/groovy/com/autonomousapps/jvm/projects/IncludedBuildWithSubprojectsProject.groovy

```diff
@@ -107,8 +109,12 @@ final class IncludedBuildWithSubprojectsProject extends AbstractProject {
     return actualProjectAdvice(project)
   }
 
-  final Set<ProjectAdvice> expectedIncludedBuildHealth = [
-    projectAdviceForDependencies(':second-sub1', [] as Set<Advice>),
+  final Set<ProjectAdvice> expectedIncludedBuildHealth(String buildNameInAdvice) {[
+    projectAdviceForDependencies(':second-sub1', [
+      useProjectDependencyWherePossible
+        ? Advice.ofChange(projectCoordinates(':second-sub2', null, buildNameInAdvice), 'api', 'implementation')
+        : Advice.ofChange(includedBuildCoordinates('second:second-sub2', projectCoordinates(':second-sub2', 'second:second-sub2', buildNameInAdvice)), 'api', 'implementation')
```

Comment:
This was not covered by the tests: Inside an included build, I may refer to my own projects in two different ways:
- `project(':path')`
- `group:name`

Before this PR, it depended on if the project is in the _root_ or one of the other builds, which notation would be suggested. Now the used notation is preserved (for change/remove advices). For add advices, the  `project(':path')` notation is suggested always.

Reply:
I actually didn't know you could refer to your own projects using `group:name` coordinates.

Reply:
This is possible as soon as you "opt-in" by including at least one build.

If you want it for a "singe build" you can do `includeBuild(".")` - https://docs.gradle.org/current/userguide/composite_builds.html#included_build_declaring_substitutions

It is good to have a consistent notation if you work with included builds to split up your own product. Then you don't need different notations to refer to the same component in different places.